### ISSUE TITLE
Can't download Binary packages

### ISSUE THREAD

**Issue by Fanho27** [xmrig/xmrig#2948]
Hello,
I've been trying to dowload the binary release for windows but I keep getting failed download - Network error message. I tried with chrome and brave. 
Any idea what's causing the issue?
Thanks

**Comment by Spudz76** [comment]
Virus block, whitelist site.

**Comment by Spudz76** [comment]
May also require [disabling chrome safe junk](https://www.minitool.com/news/how-to-stop-chrome-from-blocking-downloads.html)

**Comment by toy1111** [comment]
I use Firefox. After the download fails go into the Downloads area, click on the failed/blocked download and you'll have option to ignore/continue download. No such option in Chrome. Unfortunate fact that xmrig and other mining software gets flagged as virus.

**Comment by Spudz76** [comment]
Yes that's why you have to turn Safe Browsing off completely in Chrome, to allow for deciding yourself what downloads are safe.

**Comment by pdxwebdev** [comment]
> Hello, I've been trying to dowload the binary release for windows but I keep getting failed download - Network error message. I tried with chrome and brave. Any idea what's causing the issue? Thanks

Atempt the download again. Then goto your downloads section in Chrome (ctrl+j) it will give you an option to download anyway.
